argus/action_manager/windows.py

```diff
@@ -365,6 +366,9 @@ class WindowsNanoActionManager(WindowsSever2016ActionManager):
     _DOWNLOAD_SCRIPT = "FastWebRequest.ps1"
     _COMMON = "common.psm1"
     _RESOURCE_DIRECTORY = r"C:\nano_server"
+    _CBINIT_URL = "https://www.cloudbase.it/downloads/"
+    _BASE_DIR = r"C:\Program Files\Cloudbase Solutions\Cloudbase-Init"
```

Comment:
and we control in the nano image generation tools the path where cloudbase-init is put